announcements/2021/week09.md

```diff
@@ -0,0 +1,29 @@
+# :zap: This week in provider-independent announcements (March 5, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions...
+
+[Kong v1.2.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/) upgrades Kong to v2.3...
```

Comment:
@giantswarm/app-squad-kong can you help here? there's a note that says 'important' but then I don't know what's 1.15.0, etc... what's important for customers to know here? https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/

Also, `upgrade documentation` goes to 404... https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/helm/kong-app/UPGRADE.md